### PR TITLE
feat(modules.advices): add support recommender system advices

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/sprint-2238-2240`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-9410
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. Implement Recommender System advices
2. Manage to show UP-SELL/CROSS-SELL classic advices or Recommender System Advices
3. Implement tracking

## Translation

- [x] TRANS-46631

### CODE REVIEW

**Review by ghyenne — DISMISSED** [review]

File: packages/manager/modules/advices/src/advices/controller.js [hunk 1]
```diff
@@ -30,6 +39,10 @@ export default class AdvicesCtrl {
       });
   }
 
+  fetchAllAdvices() {
+    return [this.loadAdvices(), this.loadRecommenderSystemAdvices()];
+  }
```
Comment (JeremyDec) [inline packages/manager/modules/advices/src/advices/controller.js:44]: **nitpick**
Why not directly call these promises in `$q.all`?

File: packages/manager/modules/advices/src/advices/template.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
                     data-ng-bind="advice.localizedDescription"
                 ></span>
                 <oui-tile-button
-                    data-href="{{ ::advice.href }}"
+                    data-href="{{ advice.href || advice.url }}"
```
Comment (JeremyDec) [inline packages/manager/modules/advices/src/advices/template.html:22]:
```suggestion
                    data-href="{{:: advice.href || advice.url }}"
```

File: packages/manager/modules/advices/src/advices/controller.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/advices/src/advices/controller.js:44]: To have one dedicated location to load advices.

File: packages/manager/modules/advices/src/advices/template.html (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/advices/src/advices/template.html]: done

File: packages/manager/modules/advices/src/advices/controller.js [hunk 3]
```diff
@@ -40,6 +53,73 @@ export default class AdvicesCtrl {
...
+
+  isRecommenderSystemAvailable(recommenderAdvices) {
+    const advices = recommenderAdvices?.recommendations || [];
+    const localizedAdvices = advices.filter((advice) => {
+      return !!advice.localizedDescription[this.coreConfig.getUserLocale()];
+    });
+
+    return (
+      localizedAdvices.length > 0 &&
+      ['EU', 'CA'].includes(this.coreConfig.getRegion()) &&
```
Comment (anooparveti) [inline packages/manager/modules/advices/src/advices/controller.js:82]: Can we use feature-flipping for this?

Comment (mohammed-zahaf) [reply packages/manager/modules/advices/src/advices/controller.js]: completed

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — DISMISSED** [review]

**Comment by anooparveti** [comment]
Rebasing to `release/prod-manager-w42`

**Review by anooparveti — APPROVED** [review]

**Pull request merged**